src/community/oseo/oseo-service/src/test/java/org/geoserver/opensearch/eo/SearchTest.java

```diff
@@ -976,46 +980,48 @@ public void testGenericHTMLTemplate() throws Exception {
         summaryHasLink(
                 dom,
                 oseo
-                        + "search?parentId=SENTINEL2&uid="
+                        + "search?parentId=SENTINEL2&amp;uid="
                         + uid
-                        + "&httpAccept=application%2Fatom%2Bxml");
+                        + "&amp;httpAccept=application%2Fatom%2Bxml");
```

Comment:
No, this is a regression. The HTML part is inside a CDATA block, so the URLs inside the attribute do not need the XML escaping. Please fix (this and all the other similar changes).

Reply:
fixed.